xla/stream_executor/blas.h

```diff
@@ -317,7 +327,9 @@ class BlasSupport {
                                  const DeviceMemoryBase &a, int lda,
                                  const DeviceMemoryBase &b, int ldb,
                                  const void *beta, DeviceMemoryBase *c, int ldc,
-                                 const NumericOptions &numeric_options) = 0;
+                                 const NumericOptions &numeric_options,
+                                 blas::CallContext context) = 0;
```

Comment:
Can we give the new `context` parameter a default of `kNone`? That would make this change break less code and in most cases `kNone` is what would be used.

Also for all other API changes that add this new paramter.

Reply:
@sherhut  I added default value for context parameter. Please review.
Thanks!